docs_user/modules/proc_configuring-a-ceph-backend.adoc

```diff
@@ -33,11 +33,7 @@ ceph auth caps client.openstack \
 .Prerequisites
 
 * The `OpenStackControlPlane` custom resource (CR) must already exist.
-
-.Variables
-
-Define the shell variables used in the steps below. The values are
-just illustrative, use values that are correct for your environment:
+* Define the following shell variables. The values are illustrative. Use values that are correct for your environment: 
```

Comment:
I wonder if `illustrative` might challenge a non-native English speaker? Not sure, but I think we target a 7-8th grade reading level in our docs.

Maybe this?

````
Define the following shell variables. The values used here are examples. Replace these example values with values that are correct for your environment:
````